example/ios/File.swift

```diff
@@ -0,0 +1,6 @@
+//
+//  File.swift
```

Comment:
Is this file intended?

Reply:
File.swift was autogenerated, including the comment at the top